src/pybind/mgr/pg_autoscaler/module.py

```diff
@@ -761,18 +763,29 @@ def _maybe_adjust(self) -> None:
                                    .format(p['pool_name'],
                                            p['pg_num_final'], r))
 
+        warn_pools_string = ",".join(warn_pools)
```

Comment:
nit, single quotes are used elsewhere in the file

Reply:
@nithya-kumar , let me know if you need help addressing the above change!

Reply:
Basically, you'll want to go back to your bugfix branch in your terminal and follow the same workflow to add a new commit, where you add single quotes.

See the basic workflow here: https://docs.ceph.com/en/latest/dev/developer_guide/basic-workflow/

Reply:
Hi @ljflores - I have fixed this and updated the PR. thanks!

Reply:
@nithya-kumar great job! The only thing you'll need to fix is drop the Merge commit that is now on your branch (https://github.com/ceph/ceph/pull/48151/commits/fe3b80c8f534c3d0c13fa385c080a279e692811d). This is a really easy mistake to make when you rebase your branch, but for the future, you want to make sure to only have the regular commits on the PR, no merge commits.

To get rid of it, you can go to your bugfix branch in your terminal and run:
1. `git log`
2. Copy the SHA of the commit right before the Merge commit, i.e.:

So if your `git log` looks like this:
```
commit fe3b80c8f534c3d0c13fa385c080a279e692811d (HEAD -> ghc-osd-nithya-kumar-bug-42767, nithya-kumar/ghc-osd-nithya-kumar-bug-42767)
Merge: e1a27b70b71 9fe8b49cc06
Author: Nithya Kumar <nithya.rvkmeena@gmail.com>
Date:   Tue Sep 20 10:08:26 2022 -0400

    Merge branch 'ghc-osd-nithya-kumar-bug-42767' of https://github.com/nithya-kumar/ceph into ghc-osd-nithya-kumar-bug-42767

commit e1a27b70b710916b2331ff4b0c4701b25e450e4c (origin/ghc-osd-nithya-kumar-bug-42767)
Author: Nithya Kumar <nithya.rvkmeena@gmail.com>
Date:   Fri Sep 16 13:08:07 2022 -0700

    pybind/mgr/pg_autoscaler: warning message updated to mention which pool is affected
    
    This commit updates the warning message to mention which pool is affected when using PG autoscaler in "warn" mode.
    
    Fixes: https://tracker.ceph.com/issues/42767
    Signed-off-by: Nithya Kumar <nithya.rvkmeena@gmail.com>

commit 9fe8b49cc069d1008fc29ab93d4c1290add13aa0
Author: Nithya Kumar <nithya.rvkmeena@gmail.com>
Date:   Fri Sep 16 13:08:07 2022 -0700

    pybind/mgr/pg_autoscaler: warning message updated to mention which pool is affected
    
    This commit updates the warning message to mention which pool is affected when using PG autoscaler in "warn" mode.
    
    Fixes: https://tracker.ceph.com/issues/42767
    Signed-off-by: Nithya Kumar <nithya.rvkmeena@gmail.com>
```
You'd copy the SHA `e1a27b70b710916b2331ff4b0c4701b25e450e4c`.

3. `git reset --hard <the SHA you just copied>`
4. If you run `git log`, you should see that just the one commit is at the top now, and it should include your change with the single quotes.

In the future, you don't have to merge anything to introduce a new commit to your PR. All you'd do is:
1. Go to the bugfix branch
2. Make your change (say you made a change in file `src/pybind/mgr/pg_autoscaler/module.py`).
3. `git add src/pybind/mgr/pg_autoscaler/module.py`
4. `git commit`
5. Write and save your commit
6. `git push <the name of your remote repository> <the name of your bugfix branch>`

Then, the new commit will be cleanly added to your branch with no merge commits.